consensus/service/src/api/client_api_service.rs

```diff
@@ -188,6 +188,14 @@ impl ConsensusClientApi for ClientApiService {
         result = result.and_then(|mut response| {
             let num_blocks = self.ledger.num_blocks().map_err(ConsensusGrpcError::from)?;
             response.set_block_count(num_blocks);
+
+            // This is okay because it is invalid to start consensus without an origin block
+            // anyways
+            let block = self
+                .ledger
+                .get_block(num_blocks - 1)
+                .map_err(ConsensusGrpcError::from)?;
+            response.set_block_version(block.version);
```

Comment:
This seems wrong to me. We could be on version 3 even if we haven't produced a version 3 block yet (e.g. when we are about to transition from 2 to 3).

Reply:
i see, i guess you are right. will fix